### DISCUSSION TITLE
How to enable toolbar button on Firefox?

### DISCUSSION THREAD

**Discussion by bobdrad** [darkreader/darkreader#11146] [Q&A]
Isn't there supposed to be a button in Firefox that can be added to the toolbar to toggle Dark Reader on/off (either globally or for the current site)? I don't see it.

No button for Dark Reader is shown. Customize Toolbar shows no button for it which can be added. I can use several clicks to get into the Extension management screen, but I am looking for the toolbar button to manage it directly.

Firefox 112.0 (64-bit)
Ubuntu 22.04
Dark Reader 4.9.63 April 17, 2023

This is a relatively fresh installation of Firefox (and Ubuntu) with no significant tweaks, and a fresh installation of the Extension.

Any tips appreciated!

**Comment by ngoclong19** [comment]
You can find Pin to toolbar here, click on the gear icon:
![ảnh](https://user-images.githubusercontent.com/2577653/233765677-9af02a01-25cb-4066-9f76-34b72ee2bec8.png)
